src/components/PageLevelBanner/PageLeveBanner.tsx

```diff
@@ -0,0 +1,143 @@
+import clsx from 'clsx';
+import React, { ReactNode } from 'react';
+import styles from './PageLeveBanner.module.css';
+import Button from '../Button';
+import Heading, { HeadingElement } from '../Heading';
+import Icon from '../Icon';
+import Text from '../Text';
+
+export type Variant = 'brand' | 'success' | 'warning' | 'error';
+
+export type PageLeveBannerProps = {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * The description/body text of the banner
+   */
+  description?: ReactNode;
+  /**
+   * The element the description renders as
+   */
+  descriptionAs?: 'p' | 'span';
+  /**
+   * Callback when banner is dismissed. When passed in, renders banner with a close icon in the top right.
+   */
+  onDismiss?: () => void;
+  /**
+   * The title/heading of the banner
+   */
+  title?: ReactNode;
+  /**
+   * The element the title renders as
+   */
+  titleAs?: HeadingElement;
+  /**
+   * Stylistic variations for the banner type.
+   * - **brand** - results in a purple banner
+   * - **success** - results in a green banner
+   * - **warning** - results in a yellow banner
+   * - **error** - results in a red banner
```

Comment:
I like this, might steal in the future

Reply:
This is copied over from `Banner` 🤷

<img width="1177" alt="meme with text that says, middle school: 'plagiarism is unacceptable', high school: 'plagiarism is unacceptable', university: 'plagiarism is unacceptable', work: programmers, then there are two drawings of blonde guys with beards, the first says 'man, I stole your code' and the second one says 'its not my code'" src="https://user-images.githubusercontent.com/7761701/173880736-bcdb72e2-94cc-4554-b7b6-4408812896dc.png">